frontend/src/pages/org/archived-item-qa/ui/text.ts

```diff
@@ -29,33 +29,47 @@ function renderDiff(
           aria-labelledby="crawlTextHeading"
         >
           ${diff.map((part) => {
-            return html`
-              <span
-                class=${part.added
-                  ? removedText
-                  : part.removed
-                    ? addedText
-                    : ""}
+            if (part.added) {
+              return html`<del
+                aria-label="${msg("Missing text on crawl")}"
+                class="${removedText}"
+                >${part.value}</del
+              >`;
+            } else if (part.removed) {
+              return html`<ins
+                aria-label="${msg("Added text on crawl")}"
+                class="${addedText}"
+                >${part.value}</ins
+              >`;
+            } else {
+              return html`<span aria-label="${msg("Identical text")}"
                 >${part.value}</span
-              >
-            `;
+              >`;
+            }
           })}
         </div>
         <div
           class=${tw`flex-1 overflow-hidden border-dashed p-4 first-of-type:border-r`}
           aria-labelledby="qaTextHeading"
         >
           ${diff.map((part) => {
-            return html`
-              <span
-                class=${part.added
-                  ? addedText
-                  : part.removed
-                    ? removedText
-                    : ""}
+            if (part.added) {
+              return html`<ins
+                aria-label="${msg("Added text on analysis")}"
```

Comment:
Curious if this would get too verbose if there's a lot of insertions/deletions. This is an interesting approach: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ins#accessibility_concerns

Reply:
I did see that article, but chose to implement it this way instead for a few reasons:
1. It's very normal to not listen to the whole message when using a screen reader!  This is why the label starts with "added", "removed", or "identical" allowing users to easily skip to the block of text, or skip through twice to get to the next block of text.
2. "on analysis" and "on crawl" were helpful for me when navigating without vision to reinforce what part I was looking at and where I am in the navigation tree.  Also because of #1, users who are already aware will likely skip pas this as soon as they hear "added".  It's a feature!!

That said, I'd be willing to shorten to `Added text: analysis`?  Try it out!

Reply:
Changed to the colon